modules/python/tests/test_python_logmsg.c

```diff
@@ -256,6 +256,36 @@ Test(python_log_message, test_python_logmessage_get)
   PyGILState_Release(gstate);
 }
 
+Test(python_log_message, test_python_logmessage_get_as_str)
+{
+  LogMessage *msg = log_msg_new_empty();
+
+  PyGILState_STATE gstate;
+  gstate = PyGILState_Ensure();
+  {
+    cfg_set_version_without_validation(configuration, VERSION_VALUE_4_0);
+    PyObject *msg_object = py_log_message_new(msg, configuration);
+    PyDict_SetItemString(_python_main_dict, "test_msg", msg_object);
+
+    log_msg_set_value_by_name_with_type(msg, "a", "14", -1, LM_VT_INTEGER);
```

Comment:
It would be nice to see cases for all of our different kinds of value types in the `get()` and `get_as_str()` unit tests.

Reply:
I'm adding some more test cases.